### PR TITLE
prepare 1.1.4 nappy

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

**Comment by ReimarBauer** [comment]
I am in contact with the maintainer of the pypi package, next release will include LICENSE file

**Comment by ReimarBauer** [comment]
@igortg Can I get a feedstock please. ;)

**Comment by igortg** [comment]
I'm not an official reviewer. Let's ping @CJ-Wright so it can merge that.

File: recipes/nappy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win, py3]
```
Comment (CJ-Wright) [inline recipes/nappy/meta.yaml]: Does this syntax work?
I would put either `[win or py3k]` or `[win and py3k]` depending on what you are after.

File: recipes/nappy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+    - setuptools
+  run:
+    - numpy
+    - cdat-lite
+    - python
+
+test:
+  imports:
+    - nappy
```
Comment (CJ-Wright) [inline recipes/nappy/meta.yaml]: Can we get a few more tests (even more import tests are ok).

File: recipes/nappy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win, py3]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (CJ-Wright) [inline recipes/nappy/meta.yaml]: Would it be possible to pip? https://conda-forge.org/docs/meta.html#use-pip

**Comment by ReimarBauer** [comment]
@CJ-Wright done ?

**Comment by CJ-Wright** [comment]
ping @conda-forge/staged-recipes

**Comment by ocefpaf** [comment]
LGTM. Thanks @igortg and @CJ-Wright for the review!

PS: @igortg should have merge rights here now.

**Pull request merged**